.docker/qgis3-build-deps.dockerfile

```diff
@@ -97,7 +97,7 @@ RUN  apt-get update \
     mock \
     future \
     termcolor \
-  && apt-get autoremove -y python3-pip \
```

Comment:
CC @3nids I believe you wrote this file, is the removal of the removal ok for you?

Reply:
I removed the removal to highlight the issue.

Reply:
So this was temporary and you will remove the remove of the removal again?

Reply:
Yes :)

Reply:
Due to the `apt-get autoremove -y python3-pip`, the `python3-idna` package is removed :

```
The following packages will be REMOVED:
  python-pip-whl python3-asn1crypto python3-cffi-backend python3-crypto
  python3-cryptography python3-idna python3-keyring python3-keyrings.alt
  python3-pip python3-secretstorage python3-setuptools python3-wheel
  python3-xdg
```

But the `owslib` raise an error then:

```
Traceback (most recent call last):
  File "/root/QGIS/tests/src/python/test_qgsserver_wms.py", line 34, in <module>
    from owslib.wms import WebMapService
.....
ModuleNotFoundError: No module named 'idna'
CMake Error at PyQgsServerWMS.cmake:15 (MESSAGE):
```

So @3nids, do you agree about removing the autoremove command?